tests/README.md

```diff
@@ -29,23 +34,23 @@ The behavior of the `test.sh` can also be influenced by environment variables:
  - `PACKCC` - Path to compiled packcc binary. If unset, the script will compile it before running the tests.
  - `CC` - Compiler to use to compile `packcc` (if necessary) and tested programs. Defaults to `cc`.
 
-## How to write a generic test case
+## How to write generic test cases
 
-To create a new test case, just follow these simple steps:
+To create a new tests, just follow these simple steps:
```

Comment:
In this document, "test cases" and "tests" appear.
What's the difference between their definitions?

Reply:
Well, it would probably be better to use "test suite" instead of "test" here, but it is kind of long... So I used "test" to refer to a directory of tests, e.g. "tests/strings.d" would be one test.

By "test case" I meant one particular input, tested in the "test" (or "test suite"). E.g.: "test/strings.d/input-utf8.txt" would be one "test case".

If it doesn't sound correct or if you have better idea how to make the terminology more clear, feel free to edit the README.

Reply:
OK, I'll consider a better expression after merging this PR.